web/api/query.go

```diff
@@ -46,9 +47,9 @@ func httpJSONError(w http.ResponseWriter, err error, code int) {
 	fmt.Fprintln(w, ast.ErrorToJSON(err))
 }
 
-func parseTimestampOrNow(t string) (clientmodel.Timestamp, error) {
+func parseTimestampOrNow(t string, nower utility.Time) (clientmodel.Timestamp, error) {
```

Comment:
What is actually the advantage of using `utitility.Time`? The same could be achieved by just passing `now func() time.Time`. Doesn't need a new custom time and seems to be a pattern common in other projects already.